pkg/operator/options/suite_test.go

```diff
@@ -228,13 +121,7 @@ var _ = Describe("Options", func() {
 				"--kubelet-bootstrap-token", "flag-bootstrap-token",
 				"--ssh-public-key", "flag-ssh-public-key",
 			)
-			//Expect(err).To(HaveOccurred()) // TODO: Add back when karpenter-global-settings (and merge logic) are completely removed
-
-			// TODO: Remove below when karpenter-global-settings (and merge logic) are completely removed
-			Expect(err).ToNot(HaveOccurred())
-			ctx = settings.ToContext(ctx, &settings.Settings{})
-			Expect(func() { opts.MergeSettings(ctx) }).To(Panic())
-
+			Expect(err).To(HaveOccurred())
```

Comment:
Let's get in the habit of matching specific errors, as it prevents a large class of easily masked issues.
(We have a lot of this elsewhere, just taking an opportunity to improve it here since we touch this code.)

```suggestion
			Expect(err).To(MatchError(ContainSubstring("missing field, cluster-name")))
```
(and let's remove panic from description)